app/views/ladder/LadderPlayModal.coffee

```diff
@@ -27,6 +27,7 @@ module.exports = class LadderPlayModal extends ModalView
 
   initialize: (options, @level, @session, @team) ->
     @otherTeam = if @team is 'ogres' then 'humans' else 'ogres'
+    @otherTeam = 'humans' if @level.isType('ladder')
```

Comment:
We do want `@team` and `@otherTeam` to both be `'humans'`, since in this level type, there's only one team (humans), and when you are on that team, your opponent team is also humans.